apps/api/infra/api.ts

```diff
@@ -137,6 +137,11 @@ export const serviceSetup = (services: {
       },
       FINANCIAL_STATEMENTS_INAO_TOKEN_ENDPOINT:
         'https://login.microsoftonline.com/05a20268-aaea-4bb5-bb78-960b0462185e/oauth2/v2.0/token',
+      ELECTRONIC_REGISTRATION_STATISTICS_API_URL: {
+        dev: 'https://gw-api-staging.skra.is/business/tolfraedi',
+        staging: 'https://gw-api.skra.is/business/tolfraedi',
```

Comment:
Don't we want to use the staging url for staging instead of the prod url?

Reply:
I wasn't sure but yeah, probably for the best to have staging on staging 👍

Reply:
I changed it :)